contrib/tvmop/basic/ufunc.py

```diff
@@ -48,3 +50,71 @@ def vadd_gpu(dtype, ndim):
     s[C].bind(bx, tvm.thread_axis("blockIdx.x"))
     s[C].bind(tx, tvm.thread_axis("threadIdx.x"))
     return s, [A, B, C]
+
+
+def assign_by_req(a, req):
```

Comment:
move to sth like common.py?

Reply:
Shall we use the existing contrib/tvmop/utils.py or create a contrib/tvmop/basic/common.py?

Reply:
utils.py is fine

Reply:
Moved.